bin/lambda-deploy.sh

```diff
@@ -3,7 +3,7 @@
 set -e
 
 echo 'REMOVING DEV DEPENDENCIES'
-npm prune --production
+yarn --production
```

Comment:
Should this be `yarn install`? or does `yarn` default to install?

Reply:
Yep, default command is install.